src/wp-includes/class-wpdb.php

```diff
@@ -1479,6 +1479,8 @@ private function _escape_identifier_value( $identifier ) {
 	 *              Check support via `wpdb::has_cap( 'identifier_placeholders' )`.
 	 *              This preserves compatibility with `sprintf()`, as the C version uses
 	 *              `%d` and `$i` as a signed integer, whereas PHP only supports `%d`.
+	 * @since 6.3.0 Added `%...d` and `%...s` to work with the `IN()` operator, e.g.
```

Comment:
`6.4.0`?

Reply:
haha, erm, yes... I might be running a bit behind :-P